can_bus/json/BMS/BMS_tx.json

```diff
@@ -9,6 +9,15 @@
             },
             "State": {
                 "enum": "BmsState"
+            },
+            "Soc": {
+                "resolution": 0.1,
+                "min": 0,
+                "max": 100,
+                "unit": "%"
+            },
+            "SocCorrupt": {
+                "bits": 1
```

Comment:
Added to Bms_Vitals as we have hit the limit on the # of messages the BMS can send before hitting tx overflow, will need to address this